Source/WebCore/css/parser/CSSPropertyParserConsumer+Image.cpp

```diff
@@ -266,14 +267,19 @@ static RefPtr<CSSValue> consumeDeprecatedGradient(CSSParserTokenRange& range, co
 
 enum class SupportsColorHints : bool { No, Yes };
 
+static Markable<CSS::Color> consumeStopColor(CSSParserTokenRange& range, const CSSParserContext& context)
```

Comment:
I thought we usually use `std::optional` right up until we store in a `Markable` data member?